### DISCUSSION TITLE
Running on a Pi4 aarch64 - MongoDBV AVX Requirement

### DISCUSSION THREAD

**Discussion by arminus** [SquirrelCorporation/SquirrelServersManager#35] [Show and tell]
Out of the box, the compose install will not work on a Pi4/Cortex-A72/aarch64 since the latest mongo image won't start.

Using image: mongo:5.0 seems to be ok, though.

**Comment by SquirrelDeveloper** [comment]
Backlogged but not a priority for now :-)
